src/node_report.cc

```diff
@@ -189,6 +188,109 @@ unsigned int ProcessNodeReportVerboseSwitch(const char* args) {
   return 0;  // Default is verbose mode off
 }
 
+void SetVersionString(Isolate* isolate) {
+  Nan::TryCatch trycatch;
+  Nan::MaybeLocal<String> processProp = Nan::New<String>("process");
+  if (processProp.IsEmpty() || trycatch.HasCaught()) {
+    trycatch.Reset();
```

Comment:
.Reset() is the default behaviour unless .ReThrow() is called, you don't need to call it explicitly.  You also don't really need to check for .HasCaught() because .IsEmpty() is always true in case of exception.

A more succinct way of writing it:
```c++
Local<String> process_prop;
if (!Nan::New<String>("process").ToLocal(&process_prop)) return;
```